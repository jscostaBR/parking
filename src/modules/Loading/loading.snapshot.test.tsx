import * as React from 'react'
import { Loading } from '.'
import renderer from 'react-test-renderer'

describe('render Loading', () => {
  it('render correctly with type: registry', () => {
    const tree = renderer.create(
      <Loading type='registry' />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render correctly with type: confirmation', () => {
    const tree = renderer.create(
      <Loading type='confirmation' />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
