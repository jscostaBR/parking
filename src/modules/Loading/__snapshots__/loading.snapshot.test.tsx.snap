// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Loading render correctly with type: confirmation 1`] = `
<div
  className="sc-bczRLJ cKAijv"
>
  <img />
  <span>
    Confirmando...
  </span>
</div>
`;

exports[`render Loading render correctly with type: registry 1`] = `
<div
  className="sc-bczRLJ cKAijv"
>
  <img />
  <span>
    Registrando...
  </span>
</div>
`;
